Lib/importlib/_bootstrap.py

```diff
@@ -1306,6 +1306,7 @@ def _sanity_check(name, package, level):
 _ERR_MSG_PREFIX = 'No module named '
 
 def _find_and_load_unlocked(name, import_):
+    sys.audit("import", name, None, sys.path, sys.meta_path, sys.path_hooks)
     path = None
```

Comment:
It could be marginally clearer to define `path = None` before the `sys.audit()` call, to allow using a named variable instead of `None` in the arguments (I had to consult the table to remind myself what the second argument is!).

However, the change is not strictly needed & could be seen as slightly uglier.

```suggestion
    path = None
    sys.audit("import", name, path, sys.path, sys.meta_path, sys.path_hooks)
```